### ISSUE TITLE
Do you plan to add options to customize the menu icon?

### ISSUE THREAD

**Issue by bunnygirl-wyz** [kppleos/org.kpple.kppleMenu#2]
Hello there @rborgniet , how are you?
- Do you plan to add options to customize the menu icon?
- Do you plan to add for languages support?

Thank you!

**Comment by rborgniet** [comment]
Hi @bunnygirl-wyz I'm fine and you? 

Yes, I'm working on a control panel for the menu. Yeah I'm looking for a solution to adapt the menu language for everyone, stay tuned and thanks again for the support.

**Comment by bunnygirl-wyz** [comment]
I’m great too. ☺
Ok! I'm going to wait for new updates.
Thank you!

**Comment by odelucca** [comment]
In the meantime, there are any tweaks to change the icon whil you don't develop this feature @rborgniet ?

**Comment by rborgniet** [comment]
Hi, @odelucca Yes it's possible! Just change the default KDE logo, the menu is based on the default distribution logo.

**Comment by odelucca** [comment]
@rborgniet Nice :)

Can you explain how I can change it? Where it's file is located? I am new to KDE.

**Comment by rborgniet** [comment]
@odelucca  it depends on your distro... You can do a google search like "Change the startup icon to <your distro>"...
It's easy when you find the path ;)

**Comment by odelucca** [comment]
> @odelucca  it depends on your distro... You can do a google search like "Change the startup icon to <your distro>"...
> It's easy when you find the path ;)

Thanks. Actually, I've found the path. I am using McMojave theme and the start icon is located at `~/.local/share/plasma/desktopthemes/McMojave/icons/start.svgz`

But, I don't know why, if I change that icon it simply stops working. Even if I properly compress it to svgz.

There is a fallback icon, if I remove it, which is the default KDE icon. I've already googled it but can't find where it is. Do you know where the default KDE start icon is located?

**Comment by Kandelai** [comment]
> > @odelucca  it depends on your distro... You can do a google search like "Change the startup icon to "...
> > It's easy when you find the path ;)
> 
> Thanks! In McMojave theme it is located at `~/.local/share/plasma/desktopthemes/McMojave/icons/start.svgz`.
> 
> I've replaced it with Manjaro logo from `/usr/share/plasma/desktopthemes/breath-color/icons/start.svgz`

I tried this solution on my Manjaro stable release, but I am still getting the ugly KDE logo. So to the developer, I will await your update before using. I just can't stomach looking at that logo, LOL.

**Comment by rborgniet** [comment]
@Kandelai Just change the logo in `/usr/share/plasma/desktopthemes/breath-color/icons/start.svgz`

**Comment by Kandelai** [comment]
@rborgniet The logo in  /usr/share/plasma/desktopthemes/breath-color/icons/start.svgz (which is the manjaro logo) is the logo I want to appear, but instead the KDE logo appears in the toolbar.

**Comment by odelucca** [comment]
@Kandelai are you copying the `svgz` icon? (take a look on the `z` letter in the end)

**Comment by Kandelai** [comment]
@odelucca Yes I am am. I can even see see (preview) the Manjaro green logo as start.svgz. When I used your method of pasting that into ~/.local/share/plasma/desktopthemes/McMojave/icons/start.svgz to over-right the ugly kde logo, it appeared as the Manjaro logo. I thought ...ewww success, this will work, until I rebooted and it was still KDE logo in the toolbar. I even uninstalled Kpplemenu and reinstalled, same, ugly kde logo.

**Comment by odelucca** [comment]
@Kandelai strange :thinking:

I got some errors like this, but only when I tried to create the `start.svgz` directly with Inkscape. By copying it from the Manjaro folder it worked.

Sorry, but I have no idea how to help :/

**Comment by Kandelai** [comment]
@odelucca That is exactly what I did, copied it originally from the Manjaro folder. I did not create with or use inkscape. Yeay, I am at a loss. My inner-Linux-intuition is telling me it could be cached somewhere, or it is getting the start.svgz from some other theme folder I haven't found yet. Regardless, I will wait for the dev's update to change this in the UI.

**Comment by txhammer68** [comment]
you can change the icon to any icon within your current icon theme
change the metadata.desktop file within the widget folder
Icon=linux
find icons names using cuttlefish icon browser, paste that into the metadata.desktop
save and logoff/logon to to load new icon
